test/unit/lib/logger.test.js

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const sinon = require('sinon')
+const proxyquire = require('proxyquire').noPreserveCache()
+const EventEmitter = require('events').EventEmitter
+
+tap.test('Bootstrapped Logger', function (harness) {
+  harness.autoend()
+
+  let fakeLoggerConfigure
+  let fakeStreamPipe
+  let fakeLogger
+  let fakeEmitter
+  let fakeEmitterSpy
+  let fakeFS
+  let originalConsoleError
+
+  harness.beforeEach(function () {
+    // Make sure we don't pollute our logs
+    originalConsoleError = global.console.error
+    global.console.error = sinon.stub()
+
+    fakeLoggerConfigure = sinon.stub()
+    fakeStreamPipe = sinon.stub()
+
+    fakeLogger = sinon.stub().returns({
+      configure: fakeLoggerConfigure,
+      pipe: fakeStreamPipe
+    })
+
+    fakeEmitter = new EventEmitter()
```

Comment:
I wouldn't call this emitter fake, it's a real EventEmitter. Maybe `testEmitter`? I think `fake` is a good prefix for everything that sinon faked out.